### DISCUSSION TITLE
Code command launching window but is stuck on VScode icon

### DISCUSSION THREAD

**Discussion by PeterDaGrape** [blinksh/blink#1590] [Support]
I’m currently running iPadOS16 beta 8, and blink.sh launches fine but issuing the ‘code’ command will prompt for me to install the extension, upon pressing enter the window will pop up but will only show the VSCode logo, and nothing will happen, only after restarting will it open properly, and even then it will not load the extensions bar, and then I cannot do anything, the only thing that will happen is that it prompts me to login to GitHub.

**Comment by carloscabanero** [comment]
Hi! Please follow the guide on https://docs.blink.sh/advanced/code. Ie, start code with `code .` instead of just `code`. When you launch `code` without a path, it will try to load our GH project with instructions, and that may be a bit confusing because it requires you to log in to GitHub first, which is not what you expect.
